tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -101,5 +101,6 @@
     <script src="{% static 'js/jsonrpc.js' %}"></script>
 
     <script src="{% static 'telemetry/js/testing/breakdown.js' %}"></script>
+    <script src="{% static 'telemetry/js/testing/utils.js' %}"></script>
```

Comment:
Move `utils.js` above `breakdown.js` b/c if for some reason the functions in breakdown.js execute first they will fail with undefined references.

Reply:
Done